addons/website_sale/static/tests/tours/website_sale_product_reviews_reactions_public.js

```diff
@@ -0,0 +1,30 @@
+import { registry } from "@web/core/registry";
+
+registry
+    .category('web_tour.tours')
+    .add('website_sale_product_reviews_reactions_public', {
+        url: '/shop?search=Storage Box Test',
+        steps: () => [
+            {
+                trigger: '.oe_product_cart a:contains("Storage Box Test")',
+                run: "click",
+            },
+            {
+                trigger: '.o_product_page_reviews_title',
+                run: "click",
+            },
+            {
+                trigger: "#chatterRoot:shadow .o-mail-Message-textContent:contains(Bad box!)",
+                run: "hover && click",
+            },
+            {
+                trigger: "#chatterRoot:shadow .o-mail-Message .o-mail-Message-actions",
```

Comment:
Note that `.o-mail-Message` is not useful: could directly go to `.o-mail-Message-actions` from the `:shadow` (shorter selector is better)